apps/web-connect/src/features/header/Header.tsx

```diff
@@ -4,12 +4,17 @@ import {useAccount} from "wagmi";
 import {DiscordIcon, TelegramIcon, XaiLogo, XIcon} from "@sentry/ui/src/rebrand/icons/IconsComponents";
 import Burger from "@/features/header/Burger";
 import MobileNavbar from "@/features/header/MobileNavbar";
-import {useState} from "react";
+import { useState} from "react";
 
 export function Header() {
 	const {open} = useWeb3Modal();
 	const {address} = useAccount()
 	const [isNavbarOpened, setIsNavbarOpened] = useState(false)
+	
+	function handleConnectClick() {
+		open();
+	}
+
```

Comment:
Note: calling open directly from the button click was causing an error. 

Moving the call into it's own function removed the error.